xarray/core/options.py

```diff
@@ -292,7 +292,11 @@ def __exit__(self, type, value, traceback):
 
 def get_options():
```

Comment:
If you are up for it, you could add type hints here. Not sure exactly what but something along:
`def get_options() -> FrozenDict[str, Any]:`

Reply:
Sure, I'll look into it!